src/guild/Connection.ts

```diff
@@ -135,10 +135,7 @@ export class Connection extends EventEmitter {
      * @internal
      */
     public disconnect(): void {
-        this.channelId = null;
```

Comment:
Oh, I didn't mean to remove this, I didn't notice you actually already did it, just revert this and I'll check again. Thought it was not yet done